projects/1.20/assets/project-expansion/projectexpansion/lang/zh_cn.json

```
@@ -0,0 +1,585 @@
+	"item.projectexpansion.white_matter": "白物质",
+	"item.projectexpansion.fading_matter": "渐消物质",
+	"item.projectexpansion.matter_upgrader.tooltip": "就地升级收集器、花盆栽罐和继电器",
+	"item.projectexpansion.matter_upgrader.tooltip2": "必须已经学习了下一级，并拥有足够的EMC差价来升级",
+	"item.projectexpansion.matter_upgrader.tooltip_creative": "上述限制不适用于创造模式。",
+	"item.projectexpansion.matter_upgrader": "物质升级器",
+	"item.projectexpansion.matter_upgrader.not_owner": "该方块不属于你。",
+	"item.projectexpansion.matter_upgrader.max_upgrade": "该方块已经是最高等级了。",
+	"item.projectexpansion.matter_upgrader.not_learned": "你必须先学习“%s”才能升级。",
+	"item.projectexpansion.matter_upgrader.not_enough_emc": "你没有足够的EMC来升级，需要%s。",
+	"item.projectexpansion.matter_upgrader.done": "升级完成，已消耗%sEMC。",
+	"item.projectexpansion.matter_upgrader.done_creative": "升级完成，创造模式不消耗EMC。",
+	"item.projectexpansion.arcane_tablet": "炼金术桌",
+	"item.projectexpansion.final_star_shard": "终极之星碎片",
+	"item.projectexpansion.final_star_shard.tooltip": "合成物品",
+	"item.projectexpansion.final_star": "终极之星",
+	"item.projectexpansion.final_star.tooltip": "只能拿来炫耀，未来的用途还在计划中。",
+	"item.projectexpansion.infinite_fuel": "“无限”燃料",
+	"item.projectexpansion.infinite_fuel.tooltip": "此物品不会被消耗，燃料将消耗你的EMC进行烧练。",
+	"item.projectexpansion.infinite_fuel.not_enough_emc": "没有足够的EMC来使用它，需要%s。",
+	"item.projectexpansion.infinite_steak": "“无限”牛排",
+	"item.projectexpansion.infinite_steak.tooltip": "此物品不会被消耗，牛排将消耗你的EMC进行补充。",
+	"item.projectexpansion.infinite_steak.not_enough_emc": "没有足够的EMC来使用它，需要%s。",
+	"item.projectexpansion.knowledge_sharing_book": "知识共享之书",
+	"item.projectexpansion.knowledge_sharing_book.selected": "由%s提供的知识。",
+	"item.projectexpansion.knowledge_sharing_book.stored": "已存储知识。",
+	"item.projectexpansion.knowledge_sharing_book.self": "你无法获得自己的知识。",
+	"item.projectexpansion.knowledge_sharing_book.learned": "你已学习%s。",
+	"item.projectexpansion.knowledge_sharing_book.learned_over_100": "已学习%s个物品，但没有显示在转化桌。",
+	"item.projectexpansion.knowledge_sharing_book.learned_total": "你从%s中学习了%s个物品。",
+	"item.projectexpansion.knowledge_sharing_book.no_new_knowledge": "你什么也没学到。",
+	"item.projectexpansion.knowledge_sharing_book.no_owner": "你无法从无主的知识共享之书中获得知识。",
+	"item.projectexpansion.alchemical_book": "炼金术书",
+	"item.projectexpansion.magnum_star_ein": "一级马格南之星",
+	"item.projectexpansion.magnum_star_zwei": "二级马格南之星",
+	"item.projectexpansion.magnum_star_drei": "三级马格南之星",
+	"item.projectexpansion.magnum_star_vier": "四级马格南之星",
+	"item.projectexpansion.magnum_star_sphere": "五级马格南之星",
+	"item.projectexpansion.magnum_star_omega": "六级马格南之星",
+	"item.projectexpansion.colossal_star_ein": "一级终焉能量之星",
+	"item.projectexpansion.colossal_star_zwei": "二级终焉能量之星",
+	"item.projectexpansion.colossal_star_drei": "三级终焉能量之星",
+	"item.projectexpansion.colossal_star_vier": "四级终焉能量之星",
+	"item.projectexpansion.colossal_star_sphere": "五级终焉能量之星",
+	"item.projectexpansion.colossal_star_omega": "六级终焉能量之星",
+	"item.projectexpansion.gargantuan_star_ein": "一级庞笼能量之星",
```

```suggestion
	"item.projectexpansion.gargantuan_star_ein": "一级葛甘图安能量之星",
```
本来想用`伽刚XX`的音译的，想了想还是凑不出gargantuan的音
但是葛甘图安又不太好看（

---

卡冈图雅